.github/workflows/build.yaml

```diff
@@ -0,0 +1,25 @@
+name: Build artifacts
+# we dont support beta channel on artdefs yet...
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Download Massdriver CLI
+        uses: dsaltares/fetch-gh-release-asset@0.0.8
+        with:
+          repo: "massdriver-cloud/massdriver-cli"
+          file: "^mass-.*-linux-amd64\\.tar\\.gz$"
+          regex: true
+          token: ${{ secrets.ORGANIZATION_GITHUB_TOKEN }}
+
+      - name: Build
+        run: tar -xzf mass-*.tar.gz && export PATH=$PWD:$PATH && cd definitions/artifacts && for artifact in *; do echo ${artifact}; mass schema dereference ${artifact}; done
```

Comment:
Ahh nice, no publish here but we're still checking what we can with `dereference` 👍

Reply:
Any checks we can, 

This would be a great place to also (maybe on just a pr open, not merge) check three random artifacts against the def to see if it breaks.

Reply:
Three random _prod-like_ not necessarily real artifacts.